src/platform/desktop_notifications/desktopnotify.cpp

```diff
@@ -43,14 +44,24 @@ DesktopNotify::DesktopNotify(Settings& settings_)
     connect(&notifyCore, &Snore::SnoreCore::notificationClosed, this, &DesktopNotify::onNotificationClose);
 }
 
+QString DesktopNotify::sanitize_text_for_notifications(const QString& input_text)
+{
+    QString output_text = QString(input_text);
+    output_text.replace(QRegularExpression("[^a-zA-Z0-9 _:]"), "_"); // allow only a A-Z and 0-9 and "_" and " " and ":" chars
```

Comment:
this will destroy notifications for non-latin-script languages. but looks safe.

Reply:
we need to be strict with giving random strings to commandline programs